### DISCUSSION TITLE
2.1.744 Adjusted Default Style (Seed Breaking Change)

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/Fooocus#793] [Announcements]
Before 2.1.739, Fooocus's default (`run.bat`) SDXL configuation and GPT has a minor bias towards painterly style and has a tendency to make "digital paintings". At the same time, the default cinematic style has a minor bias towards cinematic photo, with many background blur and bokeh. As a result, many photos become a bit paintly, and many paintly arts becomes a bit cinematic - we view this as a problem rather than a feature

Though some people may like these - they are too much for a default configuation. After 2.1.744, the GPT2's bias toward painterly style is removed, and the "cinematic slighty" is renamed to "Fooocus cinematic" (you can still use it). The new default becomes "Fooocus Sharp"+"Fooocus Enhance", which are recent popular prompts borrowed from many sucessful image posts from CivitAI.

2.1.744's default is technically more "default" in styles.

**All below results are using official SDXL without any custom model. Seed is just 31337. Images are non-cherry-picked batch.**

### woman (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.752 |
| - | - |
| ![old2](https://github.com/lllyasviel/misc/assets/19834515/7eee70d1-4b28-4108-997c-38355f4f9ad1) | ![image](https://github.com/lllyasviel/Fooocus/assets/19834515/47a78d63-2dc7-40d6-8988-6a1004940b59) |

### a handsome man (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old2](https://github.com/lllyasviel/misc/assets/19834515/caa7a280-1924-4cd6-99aa-3a1b2903c795) | ![2](https://github.com/lllyasviel/Fooocus/assets/19834515/55d60a25-ed83-4210-8228-7725e6cebad7) |

### modern sports car (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old3](https://github.com/lllyasviel/misc/assets/19834515/8403427c-2c78-402c-bcd4-ebbd13f0fc1b) | ![3](https://github.com/lllyasviel/Fooocus/assets/19834515/b2348581-5b34-4e44-b71b-da465d4d044a) |

### dog in front of a house (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old4](https://github.com/lllyasviel/misc/assets/19834515/39e90022-779d-46d2-ae8b-75d2c06474f8) | ![4](https://github.com/lllyasviel/Fooocus/assets/19834515/ea10b0ea-9b93-4df2-b7a1-8f8d08589665) |

### house in garden (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old6](https://github.com/lllyasviel/misc/assets/19834515/71fd467e-292c-45b2-a5d6-08305875d5a4) | ![5](https://github.com/lllyasviel/Fooocus/assets/19834515/5c53aa69-2fa1-492d-97a5-33ca9c5874b7) |

### Socrates tells a story (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old7](https://github.com/lllyasviel/misc/assets/19834515/1bb9d41b-e796-42fe-a938-0d186e8c438d) | ![6](https://github.com/lllyasviel/Fooocus/assets/19834515/b9ab4d5d-a537-4d54-b6dc-d22fa37c695d) |

### fire fighting (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old8](https://github.com/lllyasviel/misc/assets/19834515/ca947d98-1c7b-4846-b50c-9f10971ee2e4) | ![7](https://github.com/lllyasviel/Fooocus/assets/19834515/e5e3e4eb-e890-4988-9b7e-ee7d9c78e336) |

### evil magician destroy the world (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old9](https://github.com/lllyasviel/misc/assets/19834515/7f30ef81-ac93-4283-899b-a424daff2989) | ![8](https://github.com/lllyasviel/Fooocus/assets/19834515/ee70f994-98a8-4b3f-a66c-38e9b30ea154) |

### the furry elves of the forest (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old11](https://github.com/lllyasviel/misc/assets/19834515/90b155ce-6ab1-415d-b30e-576d9e3c7b86) | ![9](https://github.com/lllyasviel/Fooocus/assets/19834515/d63778eb-6b82-469a-a17c-450802f3ed4b) |

### beautiful girl (Seed 31337, 16 non-cherry-picked images in a same batch)

| 2.1.739 | 2.1.749 |
| - | - |
| ![old1](https://github.com/lllyasviel/misc/assets/19834515/a1c29258-938e-44a2-94cb-002c88b55ed5) | ![10](https://github.com/lllyasviel/Fooocus/assets/19834515/4499cca0-e76d-4484-8860-d596e14b5b5e) |

The purpose of this post is for a record of a seed breaking change. This post should not be used as a comparison. (and if you do compare, remember to click on the image to see the details in bigger images since 2.1.744's photorealism is much less paintly.

**Comment by kubilaykilinc** [comment]
Ohhhh my GOD.😮Amazing. Thank you so much for your efforts. 🙏🏻❤️

**Comment by jgmlive** [comment]
These are beautiful!  If I'd like to maintain the "painterly" look on an image, would I uncheck the Fooocus Enhance and Fooocus Sharp?
Thank you

**Reply by lllyasviel** [reply]
you can reproduce similar images by unchecking the Fooocus Enhance and Fooocus Sharp and use "fooocus cinematic"
you can almostly reproduce previous images strictly using the expansion prompt in log, but you need to use multiple lines, for example if your prompt is "dog" and previous fooocus v2 expansion is "digital painting, by some one", then you can use

    dog
    digital painting, by some one

note that these are two lines.

but we do not recommend to do this. If users want painting, then one should just use

    digital painting of a dog

We do not document those to reduce historical burden of the project

**Reply by jgmlive** [reply]
Thanks very much for the info!

**Comment by lllyasviel** [comment]
hi all,
 styles are revised in 2.1.749 and some early problems related to image quality since 2.1.741 are fixed

**Comment by eric23** [comment]
![image](https://github.com/lllyasviel/Fooocus/assets/1481250/2c415ed1-f662-40e6-b467-bb78fc603b7c)
woman - 2.1.751 - seed31337 - no preset

**Reply by lllyasviel** [reply]
I can reproduce, it seems the woman section has some issue - have temporarily hide it and will take a look soon

**Reply by lllyasviel** [reply]
I think I find the cause - fixing it ...

**Reply by eric23** [reply]
some problems also occurs in upscaling

**Reply by lllyasviel** [reply]
hi problem is fixed in 2.1.752 and examples are updated - thanks for catching this

**Comment by lllyasviel** [comment]
hi everyone,

getting good skin texture from SDXL is usually difficult without using custom models.

A good news is that Fooocus 2.1.752 can get some skin even using default SDXL+Refiner.

### closeup of girl face, blue eyes (seed 31337, 16 non-cherry-picked images in a same batch, pure default config without using any custom models)

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/1dfcea8d-710b-4cd4-9a9f-abe4bae3d28d)

some outputs

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/8c8b806a-2030-450c-983a-e76ebe46d4a3)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/c13f9ef2-d77b-4d06-89e5-c0691416f306)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/6fd633c8-71f6-45c8-9e98-aa5124387760)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/fb7f24ca-a947-4626-b885-7ce88b1923a9)


Again this is without any custom models. [Apparently](https://github.com/lllyasviel/Fooocus/discussions/679#realistic), results can be further improved with custom models.
